impexp-client/src/main/java/org/citydb/modules/kml/gui/preferences/GeneralPanel.java

```diff
@@ -362,6 +387,8 @@ public void loadSettings() {
 		gltfConverterBrowseText.setText(kmlExporter.getPathOfGltfConverter());
 		notCreateColladaCheckbox.setSelected(kmlExporter.isNotCreateColladaFiles());
 		embedTexturesInGltfCheckbox.setSelected(kmlExporter.isEmbedTexturesInGltfFiles());
+		exportGltfBinary.setSelected(kmlExporter.isExportGltfBinary());
+		enableGltfDracoCompression.setSelected(kmlExporter.isEnableGltfDracoCompression());
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).